### DISCUSSION TITLE
Remote bootstrap cluster for management creation and deletion?

### DISCUSSION THREAD

**Discussion by karuppiah7890** [karuppiah7890/tce-e2e-test#26] [Ideas]
This is @mohitsharma-in 's idea. 

Currently Tanzu Framework uses local kind bootstrap clusters for creating and deleting management clusters. Can we use remote kind bootstrap clusters?

Maybe use Elastic Container Service (ECS) from AWS? Gotta think on this

Do all compute or almost all compute in remote cloud - for example, use AWS to run bootstrap cluster and also the AWS management cluster

**Comment by mohitsharma-in** [comment]
[Details can be found on below ](https://github.com/vmware-tanzu/tanzu-framework/issues/2340)
